plat/qemu/qemu_sbsa/fdts/qemu_sbsa_spmc_sp_manifest.dts

```diff
@@ -101,8 +101,18 @@
 		reg = <0x00000100 0x1FFFE000 0x0 0x00001000>;
 	};
 
-   memory@2 {
-     device_type = "ns-memory";
-     reg = <0x00000100 0x00000000 0x0 0x80000000>;
-   };
- };
+  memory@2 {
```

Comment:
I thought you want to move this up to be next to region 1 :)

Reply:
Oh no, nothing was really changed. It was just a formatting thing that this got all weird.